molecule/default/tests/test_default.py

```diff
@@ -4,15 +4,91 @@
 import os
 
 # Third-Party Libraries
-import pytest
+# import pytest
```

Comment:
We should either delete this line or add a comment explaining why we are keeping it commented-out.

Reply:
Good catch.  I commented it out because I wasn't using it, but I left it in in case I ended up needing it.  Line removed in commit 6d567a9.